test/examples/test_multitask_gp_regression.py

```diff
@@ -19,9 +19,9 @@
 from gpytorch.random_variables import GaussianRandomVariable
 
 # Simple training data: let's try to learn a sine function
-train_x = Variable(torch.linspace(0, 1, 11))
-y1_inds = Variable(torch.zeros(11).long())
-y2_inds = Variable(torch.ones(11).long())
+train_x = Variable(torch.linspace(0, 1, 100))
```

Comment:
I'd actually prefer to leave it this way. This failed silently before when the kernel evaluation stuff got changed because of the checks involving root decomposition size, so leaving this larger will catch similar problems in the future.